README.md

```diff
@@ -261,6 +261,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Myjson](http://myjson.com/api) | A simple JSON store for your web or mobile app | No | No | Unknown |
 | [Nationalize.io](https://nationalize.io) | Estimate the nationality of a first name | No | Yes | Yes |
 | [OOPSpam](https://oopspam.com/) | Multiple spam filtering service | No | Yes | Yes |
+| [PageCDN](https://pagecdn.com/docs/public-api) | Public API for javascript, css and font libraries on PageCDN. | `apikey` | Yes | Yes |
```

Comment:
Remove `.` character at the end of the description and change `k` character in the `apikey` string to uppercase.

```suggestion
| [PageCDN](https://pagecdn.com/docs/public-api) | Public API for javascript, css and font libraries on PageCDN | `apiKey` | Yes | Yes |
```